CHANGES.rst

```diff
@@ -207,6 +207,9 @@ astropy.modeling
 
 - Many private attributes and methods have changed (see documentation). [#8769]
 
+- Added analytical King model (KingProjectedAnalytic1D) [#9084]
+- Added analytical King model (KingProjectedAnalytic1D) [#8208]
```

Comment:
@nluetzge - could you remove the duplicate entry here? (the one with #8208) - other than that I think this is ready for review by @nden. Thanks!